plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -274,7 +299,10 @@ func (e *Elasticsearch) gatherClusterHealth(url string, acc telegraf.Accumulator
 		return err
 	}
 	measurementTime := time.Now()
+	tags := map[string]string{"name": healthStats.ClusterName}
+
 	clusterFields := map[string]interface{}{
+		"cluster_name":          healthStats.ClusterName,
```

Comment:
If I understand correctly this is adding a new field named `cluster_name` after having changed the previous tag `cluster_name` to `name` with the exact same information. Is there any reason for this?